### DISCUSSION TITLE
Option to keep tooltip opened when hovering it

### DISCUSSION THREAD

**Discussion by fingeromer** [mantinedev/mantine#5020] [Feature requests]
A common behavior of tooltips is to allow hovering and copying the content of the tooltip.
It seems that in Mantine, to tooltip closes when to cursor leaves the `children` area, even if it tries to hover the tooltip it self.

Example:

[Tooltip _ Mantine.webm](https://github.com/mantinedev/mantine/assets/112797342/2b790e44-fcd8-4bac-b29e-84d2d05685f3)

Required behavior:

[Tooltip with inline SVG icon.webm](https://github.com/mantinedev/mantine/assets/112797342/f6e09e55-72e7-4d78-ad21-fc0688bb927b)

**Comment by rtivital** [comment]
This behavior is implemented in HoverCard component – https://mantine.dev/core/hover-card/

**Reply by mickeymaruf** [reply]
Hey @rtivital,
How to stop propagation on hover and only show on the current element?
TIA

**Comment by jhitchins88** [comment]
The HoverCard does have that behavior, but does not meet other accessibility requirements.  Tooltips according to [WAI tooltip guidelines](https://www.w3.org/WAI/WCAG22/Understanding/content-on-hover-or-focus.html#examples) should stay open while hovering over the tooltip itself:

> If pointer hover can trigger the additional content, then the pointer can be moved over the additional content without the additional content disappearing;

I too would like this behavior, either as a default or as an option.

**Comment by WPaczula** [comment]
I'm also interested in adding this feature to the tooltips. We'd like to stay accessible (the content we show contains additional information that is crucial) so we can't use the HoverCard.
